app/models/document.rb

```diff
@@ -46,6 +54,51 @@ def facet_options(facet)
     finder_schema.options_for(facet)
   end
 
+  def format_specific_metadata
```

Comment:
If you can think of a way to break it up so it explains itself that would be best, otherwise yeah a comment would be good stuff.